fuzz/fuzzing_target.cc

```diff
@@ -0,0 +1,24 @@
+#include "clay.h"
+#include <stdint.h>
+#include <stddef.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+    if (size < sizeof(Clay_String)) return 0;
```

Comment:
I'm not sure I understand what you're doing here... It seems you're ignoring data shorter than the practically arbitrary size of a `Clay_String`.